src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -5,31 +5,40 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
 @Data
```

Comment:
don't use `@Data` for entities
```suggestion
@Getter
@Setter
```